docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -378,17 +383,17 @@ The tabs below expand to show instructions for installing each Serving extension
 === "TLS wildcard support"
 
     If you are using a Certificate implementation that supports provisioning
-    wildcard certificates (e.g. cert-manager with a DNS01 issuer), then the most
+    wildcard certificates (for example, cert-manager with a DNS01 issuer), then the most
     efficient way to provision certificates is with the namespace wildcard
-    certificate controller. The following command will install the components needed
-    to provision wildcard certificates in each namespace:
+    certificate controller. Install the components needed to provision wildcard certificates in each
+    namespace by running the command:
```

Comment:
I'd make this a new line starting at "Install..."

Reply:
I also noticed this. It's part of my suggestions below if you want to apply this change.